onnxruntime/core/framework/node_unit.cc

```diff
@@ -351,6 +351,13 @@ void NodeUnit::InitForSingleNode() {
                                      NodeUnitIODef::QuantParam{*input_defs[1],
                                                                input_defs.size() == 3 ? input_defs[2] : nullptr,
                                                                axis}});
+  } else if (IsVariadicQLinearOp(qlinear_type)) {
```

Comment:
@skottmckay, can you take a quick look at this newly added code in this file?

Reply:
LGTM

Reply:
thanks!